tests/preconversion_assessment/test_format.py

```diff
@@ -251,9 +236,53 @@ def test_apply_message_transform(data, action_id, expected):
                 }
             ],
         ),
+        (
+            {
+                "actions": {
+                    "LIST_THIRD_PARTY_PACKAGES": {
+                        "messages": [],
+                        "result": {
+                            "title": "List packages not packaged by the original OS vendor.",
+                            "description": "Only packages from the original OS vendor and Red Hat will be converted.  List any other packages so the user knows they won't be converted and can choose whether they think it is safe to proceed.",
+                            "diagnosis": "",
+                            "id": "SUCCESS",
+                            "level": "SUCCESS",
+                            "remediation": "",
+                            "variables": {},
+                        },
+                    },
+                    "RHEL_COMPATIBLE_KERNEL": {
+                        "messages": [],
+                        "result": {
+                            "title": "Ensure the booted kernel is compatible with RHEL.",
+                            "description": "Check that the kernel currently loaded on the host is signed, is standard (not UEK, realtime, etc), and has the same version as in RHEL.  These criteria are designed to check whether the RHEL kernel will provide the same capabilities as the original system.",
+                            "diagnosis": "The booted kernel {{ kernel_version }} is compatible with RHEL.",
+                            "id": "SUCCESS",
+                            "level": "SUCCESS",
+                            "remediation": "",
+                            "variables": {
+                                "kernel_version": "3.10.0-1160.88.1.el7.x86_64"
+                            },
+                        },
+                    },
+                }
+            },
+            [],
+        ),
     ),
 )
 def test_transform_raw_data(data, expected):
     result = transform_raw_data(data)
-    print(pprint(result))
+    assert result == expected
+
+
+@pytest.mark.parametrize(
+    ("data", "level", "expected"),
+    (
+        ({"level": "SUCCESS"}, "SUCCESS", None),
```

Comment:
What is expected? `{}` or `None`? test is failing

Reply:
Empty dictionary 😓. Changed the return value for the function and forgot about the unit test.

Reply:
done.